frontend/src/components/home/measurements/PeriodPicker.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { MonthPicker } from "../../controls/form-controls/MonthPicker";
+import { Moment } from "moment";
+import { t } from "../../utils";
+
+interface Props {
+  from: Moment;
+  to: Moment;
+  fromChange: (value: any) => void;
+  toChange: (value: any) => void;
+}
```

Comment:
Fyi

Ik zou altijd specifieke prop name geven: `PeriodPickerProps`
Anders heb je heel veel interfaces die allemaal dezelfde naam hebben

Vinnek persoonlijk een grote: zorg dat je geen twee classes/dingen hebt die dezelfde naam hebben
Elke keer als je er zo eentje vast hebt weet je niet welk het precies is...